Packages/packages-lock.json

```diff
@@ -95,6 +95,15 @@
       "dependencies": {},
       "url": "https://packages.unity.com"
     },
+    "com.unity.recorder": {
```

Comment:
what is this for?

Reply:
I updated my computer to unity 2021.3.20f and I thought that took care of it

Reply:
I hope to check with you at friday s meeting about what  I need to uncheck in git hub desktop

Reply:
I rebased the changes and unstaged the Packages